modules/snippy/snippy.nf

```diff
@@ -26,5 +26,14 @@ process SNIPPY {
 
     """
 
+    stub:
+    ram = "${task.memory}".split(" ")[0]
+
+    """
+    echo snippy --cpus ${task.cpus} --ram ${ram} --outdir $genomeName --ref $refGbk --R1 ${genomeReads[0]} --R2 ${genomeReads[1]}
```

Comment:
@Mxrcon , for correct behavior of `echo` , it might be better to enclose the command within quotes.

```suggestion
    echo "snippy --cpus ${task.cpus} --ram ${ram} --outdir $genomeName --ref $refGbk --R1 ${genomeReads[0]} --R2 ${genomeReads[1]}"
```

Reply:
That's true, i made this mistake on all stubs, will fix that now